chess-AI/experimental-work/mcts.py

```diff
@@ -1,73 +1,161 @@
-#https://web.stanford.edu/~surag/posts/alphazero.html
+from math import sqrt
+import numpy as np
+from state_representation import get_cnn_input
 
 class Mcts:
-    def search(s,game,nnet):
 
-        if game.gameEnded(s): return  -game.gameReward(s) # returns the negative of the game reward, why negative?
+    def __init__(self):
+        self.states_visited = [] # stores fen strings of states visited
```

Comment:
If you are going to have comments here, they could be more descriptive. These kind of just restate the variable name. If you want to describe what the N or Q values are, you can do that, but otherwise these comments aren't needed.

Reply:
Also, if you do keep these comments at the end of the line, [they should be 2 spaces away from the end of the code](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments).